doc_source/constructs.md

```diff
@@ -1,104 +1,100 @@
 --------
 
- This documentation is for the developer preview release of the AWS CDK\. Do not use this version of the AWS CDK in production\. Subsequent releases of the AWS CDK will likely include breaking changes\. 
+ This documentation is for the developer preview release of the AWS CDK\. Do not use this version of the AWS CDK in production\. Subsequent releases of the AWS CDK will likely include breaking changes\.
 
 --------
 
 # Constructs<a name="constructs"></a>
 
-Constructs are the building blocks of AWS CDK applications\. Constructs can have child constructs, which in turn can have child constructs, forming a hierarchical tree structure\.
+Constructs can be viewed as "cloud components". They can represent architectures of any complexity. They can represent a single resource, such as an Amazon S3 Bucket or an Amazon SNS Topic, reusable components such as a static website, a part of a specific application and up to complex multi-stack applications that span multiple accounts and region. Everything in the AWS CDK is a construct.
 
-The AWS CDK includes two different levels of constructs:
+This composition of constructs also means you can easily create sharable constructs, and make changes to any construct and have those changes available to consumers as shared class libraries\.
```

Comment:
Maybe you should briefly define the 'scope hierarchy'/'construct tree' here to explain the encapsulation and composability. Having the High/Low level explanation in between seems out of order to me.

Reply:
I've added some more explanatory text.